README.md

```diff
@@ -60,6 +60,7 @@ This sample demonstrates the following MSAL Angular concepts:
 1. Open the [/src/app/app.module.ts](./src/app/app.module.ts) file and provide the required configuration values.
     1. Replace the string `"Enter_the_Application_Id_Here"` with your app/client ID on AAD Portal.
     1. Replace the string `"Enter_the_Cloud_Instance_Id_HereEnter_the_Tenant_Info_Here"` with `"https://login.microsoftonline.com/common/"`
+    NOTE: As of 2021, common auth is not supported use `"https://login.microsoftonline.com/DIRECTORY_TENANT_ID"`
```

Comment:
@arvindn05 I'm not aware that "common" endpoint being deprecated -may I ask if you have source for this?